mabwiser/base_mab.py

```diff
@@ -83,8 +87,25 @@ def __init__(self, rng: _BaseRNG, arms: List[Arm], n_jobs: int, backend: str = N
         self.backend: str = backend
 
         self.arm_to_expectation: Dict[Arm, float] = dict.fromkeys(self.arms, 0)
-        self.cold_arm_to_warm_arm: Dict[Arm, Arm] = dict()
-        self.trained_arms: List[Arm] = list()
+        self._reset_arm_to_status()
+
+    @property
+    def cold_arm_to_warm_arm(self) -> Dict[Arm, Arm]:
+        """Mapping indicating what arm was used to warm-start cold arms."""
+        return {cold_arm: status[STATUS_WARM_STARTED_BY] for cold_arm, status in self.arm_to_status.items()
+                if status[STATUS_WARM]}
+
+    @property
+    def trained_arms(self) -> List[Arm]:
+        """List of trained arms.
+
+        Arms for which at least one decision has been observed are deemed trained."""
+        return [arm for arm in self.arms if self.arm_to_status[arm][STATUS_TRAINED]]
+
+    @property
+    def cold_arms(self) -> List[Arm]:
```

Comment:
Quick caveat here: by this definition, all arms in the neighborhood policies are cold. At a LP level, this might make sense.. but I don't think it makes sense at MAB level, so the `cold_arms` property in the MAB object has additional logic to make all arms "warm" if it's using a neighborhood policy.

Reply:
I'm not sure if it makes things any simpler for cold arms. We don't actually use the cold arms for neighborhood policies, and even if we replaced warm start with a simple `pass` in the neighborhood policy wouldn't we still need to have a definition of cold arms if we're making that available at MAB level?

Reply:
No not for cold arms specifically. But it would be more consistent to say that warm start works for all LPs (in pretty much the same way), but that NPs cannot be warm started. If there are no major objections, I'm going to bundle this change here.